lib/improver/tests/utilities/test_cli_utilities.py

```diff
@@ -30,11 +30,18 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """Unit tests for utilities.cli_utilities."""
 
+import os
+import numpy as np
 import unittest
 from unittest.mock import patch, mock_open
+from tempfile import mkdtemp
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [standard import "from tempfile import mkdtemp" should be placed before "import numpy as np"](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4683231)